.claude/commands/new-portal.md

```
@@ -0,0 +1,121 @@
+```
+
+### 3. Copy the template
+
+```bash
+cp -r "$TEMPLATE_DIR" "./$PROJECT_SLUG"
+# Remove build artifacts that must not be copied
+rm -rf "./$PROJECT_SLUG/node_modules" "./$PROJECT_SLUG/.next"
+```
+
+If the destination already exists and is non-empty:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)